packages/react/src/components/MultiSelect/__tests__/MultiSelect-test.js

```diff
@@ -3,380 +3,461 @@
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
+ *
  */
 
-import React from 'react';
-import { mount } from 'enzyme';
-import MultiSelect from '../../MultiSelect';
+import 'carbon-components/src/components/multi-select/_multi-select.scss';
+import { getByText, isElementVisible } from '@carbon/test-utils/dom';
 import {
-  openMenu,
-  generateItems,
-  generateGenericItem,
-} from '../../ListBox/test-helpers';
-import { settings } from 'carbon-components';
+  pressEnter,
+  pressSpace,
+  pressTab,
+  pressArrow,
```

Comment:
Do we need a pressArrow helper? If I remember correctly, we call out `pressEnter`/`pressSpace`/`pressTab` as they have some special behavior associated with them in the browser that we need to replicate in JSDOM.

Reply:
Ah, thought it was just a file of keyboard helpers. I'll scrap it 👍